src/Sarif.UnitTests/TestData/SarifCurrentToVersionOneVisitor/Inputs/OneRunWithRules.sarif

```diff
@@ -9,25 +9,29 @@
       },
       "results": [
         {
-          "ruleId": "C2009",
+          "ruleId": "C2001",
+          "ruleIndex":  0,
```

Comment:
>ruleIndex [](start = 11, length = 9)

why couldn't the prerelease transformer fix things up for you? i.e., was there a reason you had to introduce the ruleIndex object? generally, i think we should avoid piecemeal upgrade of these files to current format.

Reply:
`FileDiffingTests` runs the prerelease transformer on the _expected output_ (except in the case of tests that, like this one, produce SARIF v1), not on the input.

---
In reply to: [249573084](https://github.com/Microsoft/sarif-sdk/pull/1228#discussion_r249573084) [](ancestors = 249573084)